### ISSUE TITLE
Feature: Ranking

### ISSUE THREAD

**Issue by LadyNanuia** [ksandin/racp#182]
make it an option to turn on rankings for things like mvps, levels, alchemists etc

**Comment by ksandin** [comment]
I assume you mean player rankings, ie. A leaderboard feature?

You need to be more specific about each type of leaderboard. Explain how the leaderboard is sorted for each type. For example, how should an mvp leaderboard work? Most killed bosses? Do all bosses compare equally or do you gain more points for harder bosses?

Also, ”etc” is too vague. Please specify all leaderboards you want.

Also, how complex should it be? Should there be a season system with regular resets, or manual admin reset? Or should it just be one global leaderboard that collects data forever with no care about time?

**Comment by LadyNanuia** [comment]
and for mvps, show a ranking for each mvp and who has the most kills / its total deaths?

**Comment by LadyNanuia** [comment]
any updates on this? =)

**Comment by ksandin** [comment]
Nope, haven’t touched RACP in a while. Not sure when I’ll be able to either.

**Comment by LadyNanuia** [comment]
> Nope, haven’t touched RACP in a while. Not sure when I’ll be able to either.

No problem, just thought i'd ask!
